CHANGELOG.md

```diff
@@ -1,3 +1,38 @@
+## 15.1.0
+
+_released `2023-09-07`_
+
+### cli
+
+- :bug: Removed the logic converting an empty array to undefined in changelog metadata ([#690](https://github.com/zapier/zapier-platform/pull/690))
+- :nail_care: Integration title length must be at least 2 characters ([#693](https://github.com/zapier/zapier-platform/pull/693))
+
+### core
+
+None!
+
+### schema 
+
+None!
+
+### legacy-scripting-runner
+
+- :bug: Fixed a flaky test for legacy scripting runner ([#697](https://github.com/zapier/zapier-platform/pull/697))
+
+### misc
+
+- :scroll: Added links to relevant support documentation ([#692](https://github.com/zapier/zapier-platform/pull/692))
+- :scroll: Fixed markdown formatting for backtick usage ([#691](https://github.com/zapier/zapier-platform/pull/691))
+- :scroll: Added detail on encoding defaults in the API documentation ([#689](https://github.com/zapier/zapier-platform/pull/689))
+- :scroll: Added detail on OAuth state parameter in the API documentation ([#683](https://github.com/zapier/zapier-platform/pull/683))
+- :scroll: Minor updates to CLI documentation and rebuild ([#688](https://github.com/zapier/zapier-platform/pull/688))
+- :scroll: Moved up template explanation in README ([#682](https://github.com/zapier/zapier-platform/pull/682))
+- :scroll: Added detail on the 'computed' flag in the API documentation ([#686](https://github.com/zapier/zapier-platform/pull/686))
+- :scroll: Added detail on callback URL parameter in the API documentation ([#687](https://github.com/zapier/zapier-platform/pull/687))
+- :scroll: Updated GitHub CLI tutorial app documentation ([#679](https://github.com/zapier/zapier-platform/pull/679))
+- :hammer: Bumped word-wrap version from 1.2.3 to 1.2.4 ([#680](https://github.com/zapier/zapier-platform/pull/680))
+- :hammer: Bumped cli, core, schema from 15.0.0 to 15.0.1 ([#678](https://github.com/zapier/zapier-platform/pull/678))
```

Comment:
Is this list item necessary to include? 🤔

Reply:
going to merge but we can revisit this discussion async